buildSrc/src/main/kotlin/pklKotlinLibrary.gradle.kts

```diff
@@ -24,6 +24,11 @@ tasks.compileKotlin {
   enabled = true // disabled by pklJavaLibrary
 }
 
+kotlin.jvmToolchain {
+  languageVersion.set(jvmToolchainVersion)
+  vendor.set(jvmToolchainVendor)
+}
```

Comment:
This should not be necessary because this plugin applies `pklJavaLibrary`, and the latter already applies the toolchain in the `java {}` block.